package.json

```diff
@@ -144,8 +148,10 @@
     "uuid": "2.0.1",
     "valid-url": "1.0.9",
     "walk": "2.3.4",
-    "webpack": "1.13.1",
-    "webpack-dev-middleware": "1.2.0",
+    "webpack": "3.0.0",
```

Comment:
performance problem was fixed even within the 3.0 and is active in this branch :)